components/rmrk/Gallery/PaginatedCardList.vue

```diff
@@ -123,14 +134,15 @@ export default class PaginatedCardList extends mixins(
     this.isFetchingData = true
     const result = await this.$apollo.query({
       query: this.query,
-      client: this.urlPrefix,
+      client: 'subsquid',
       variables: {
         account: this.account,
-        orderBy: this.searchQuery.sortBy,
-        search: this.buildSearchParam(),
-        first: this.first,
+        orderBy: this.remapSortBy,
+        and: this.buildSearchParam,
+        limit: this.first,
         offset: (page - 1) * this.first,
       },
+      fetchPolicy: 'no-cache',
```

Comment:
?

Reply:
Apollo diff bug.
if not using 'no-cache', it would return the same data.